classes/options-general.php

```diff
@@ -858,31 +858,31 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 													<tr valign="middle">
 														<td>
 															<label
-																for="display_search"><?php _e( 'Display Search Form', 'simplemap' ); ?></label>
+																for="display_search"><?php _e( 'Display Search Form', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<label for="display_search_yes"><input type="radio"
 															                                       name="display_search"
 															                                       id="display_search_yes"
-															                                       value="show" <?php checked( $display_search, 'show' ); ?> /> <?php _e( 'Yes', 'simplemap' ); ?>
+															                                       value="show" <?php checked( $display_search, 'show' ); ?> /> <?php _e( 'Yes', 'SimpleMap' ); ?>
 															</label>&nbsp;&nbsp;
 															<label for="display_search_no"><input type="radio"
 															                                      name="display_search"
 															                                      id="display_search_no"
-															                                      value="hide" <?php checked( $display_search, 'hide' ); ?> /> <?php _e( 'No', 'simplemap' ); ?>
+															                                      value="hide" <?php checked( $display_search, 'hide' ); ?> /> <?php _e( 'No', 'SimpleMap' ); ?>
 															</label><br/>
 														</td>
 													</tr>
 
 													<tr valign="middle">
 														<td>
 															<label
-																for="powered_by"><?php _e( 'SimpleMap Link', 'simplemap' ); ?></label>
+																for="powered_by"><?php _e( 'SimpleMap Link', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<label for="powered_by"><input type="checkbox"
 															                               name="powered_by"
-															                               id="powered_by" <?php checked( $powered_by ); ?> /> <?php _e( 'Show the "Powered by SimpleMap" link', 'simplemap' ); ?>
+															                               id="powered_by" <?php checked( $powered_by ); ?> /> <?php _e( 'Show the "Powered by SimpleMap" link', 'SimpleMap' ); ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 210 characters

<!-- Comment by [SideCI](https://sideci.com) -->